### DISCUSSION TITLE
Cannot retrieve  "Evacuation-Immediate" from API

### DISCUSSION THREAD

**Discussion by formula4** [weather-gov/api#256] [Q&A]
Greetings,

I am using curl to retrieve active WWA's and I have no problems with getting any of them, except "Evacuation-Immediate".

Here is a curl request that I am using in the windows cmd prompt that works (severe thunderstorm warning as an example). I use "%%20" to be able to make the request with spaces.

`curl -k "https://api.weather.gov/alerts/active?event=Severe%%20Thunderstorm%%20Warning" > SVR.json`

However, Evacuation-Immediate does not work, and returns an error 400 : 

`curl -k "https://api.weather.gov/alerts/active?event=Evacuation%%20-%%20Immediate" > Evacuation.json`

I have tried without spaces, with spaces, without the hyphen, etc - it always returns 400. What am I doing wrong? I am hoping this is just a general learning experience in the command prompt for me and not something wrong with the API.

**Comment by JJKraw** [comment]
I noticed that it's the only event with a dash/hyphen. Tried both an en- and em-dash neither worked.

Then took a wild guess and omitted the dash altogether. Bingo.

So - it looks like the dash/hyphen should be removed from the return data for the alerts/types endpoint.

**Comment by formula4** [comment]
Thank you - that was it. I would close this issue but I will wait until someone removes it from the alerts/types endpoint.
